sdk/core/azure-core-amqp/src/amqp/connection.cpp

```diff
@@ -556,7 +554,7 @@ namespace Azure { namespace Core { namespace Amqp { namespace _detail {
     {
       throw std::runtime_error("COuld not get properties.");
     }
-    return Models::AmqpValue{value}.AsMap();
+    return Models::AmqpValue{Models::_detail::UniqueAmqpValueHandle{value}}.AsMap();
```

Comment:
I noticed a small typo two lines above, and I can't leave a "suggestion" type of GitHub comment.

`COuld` => `Could`

Reply:
It's been there for a while - I'll fix this in the next go-round.